drivers/bluetooth/hci/Kconfig

```diff
@@ -345,12 +345,14 @@ config BT_HCI_SETUP
 
 config BT_DRV_TX_STACK_SIZE
 	int
+	depends on BT_H5
 	default 256
 	help
 	  Stack size for the HCI driver's TX thread.
 
 config BT_DRV_RX_STACK_SIZE
 	int
+	depends on BT_H4 || BT_H5 || BT_SPI || BT_HCI_RAW_H4 || BT_AMBIQ_HCI || BT_STM32_IPM || HCI_NXP_RX_THREAD
```

Comment:
A more maintainable/scalable solution for something like this is to define a virtual option that you depend on, and then each driver (that needs this) selects that virtual option.

Reply:
Wouldn't it make more sense to have each of the drivers define their own Kconfig variable? Is there a reason it makes sense to have a shared variable?

Reply:
That's a fair point. Driver specific options should be fine as well.

Reply:
> Wouldn't it make more sense to have each of the drivers define their own Kconfig variable? Is there a reason it makes sense to have a shared variable?

Not sure of the history behind the common `BT_DRV_RX_STACK_SIZE`  being used, but some of the vendors already has driver specific rx stack size values defined, that is being assigned to this shared Kconfig.